src/main/java/uk/gov/hmcts/reform/profilesync/service/impl/ProfileUpdateServiceImpl.java

```diff
@@ -90,7 +90,7 @@ private  ProfileSyncAuditDetails syncUser(String bearerToken, String s2sToken,
             log.error("{}:: Exception occurred while updating the user profile: Status - {}"
                             + response.status(), loggingComponentName);
             message = "the user profile failed while updating the status";
-            log.error("{}:: Body response::{}" + response.body(), loggingComponentName);
+            log.error("{}:: Body response::{}", loggingComponentName);
```

Comment:
we can remove this logging as we are not including the body response at all in the log

Reply:
Done